csv/src/test/scala/akka/stream/alpakka/csv/CsvParserSpec.scala

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.csv
+
+import akka.stream.alpakka.csv.CsvParser.MalformedCSVException
+import akka.util.ByteString
+import org.scalatest.{Matchers, OptionValues, WordSpec}
+
+class CsvParserSpec extends WordSpec with Matchers with OptionValues {
+
+  "CSV parser" should {
+    "read comma separated values into a list" in {
+      val in = ByteString("one,two,three\n")
+      val parser = new CsvParser()
+      parser.offer(in)
+      val res = parser.poll()
+      res.value.map(_.utf8String) should be(List("one", "two", "three"))
+      parser.poll() should be('empty)
+    }
```

Comment:
Maybe a nice little abstraction would be in place here, as it is pretty much a long repetition of the same logic with different input and output? (Not necessarily something to do anything about though)